pwa/app/routes/insights.($year).tsx

```diff
@@ -195,8 +201,27 @@ function LeaderboardKeyList({
           <LeaderboardKeyLink keyType={keyType} keyVal={k} />
         </React.Fragment>
       ))}
-      {keyVals.length > cutoffSize &&
-        ` (... and ${keyVals.length - cutoffSize} more)`}
+      {keyVals.length > cutoffSize && (
+        <TooltipProvider>
+          <Tooltip>
+            <TooltipTrigger className="italic">
+              &nbsp;
+              {keyVals.length > cutoffSize &&
```

Comment:
Is this condition necessary? isnt `keyVals.length > cutoffSize` already true since it's being checked above?